metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -176,7 +176,11 @@ def make_tableau_client(self) -> Server:
             )
 
         try:
-            server = Server(self.connect_uri, use_server_version=True)
+            server = Server(
+                self.connect_uri,
+                use_server_version=True,
+                http_options={"verify": self.ssl_verify},
```

Comment:
Does this also work as expected if we set path to pem file in `ssl_verify` ? [This answer](https://community.tableau.com/s/question/0D54T00000rP9CwSAK/tableau-server-client-ssl-connection-issue) led to me to believe that we need to set separate `cert` option and that cert needs to be in same path as script.

For our usecase though, it might be okay to simply pass `verify=False` if ssl_verify is not None. WDYT ?

Reply:
Yep that's right - fixed it